tasks/main.yml

```diff
@@ -25,10 +25,11 @@
   service: name=jenkins state=started enabled=yes
 
 - name: Wait for Jenkins to start up before proceeding.
-  wait_for:
-    host: "{{ jenkins_hostname }}"
-    port: 8080
-    timeout: 300
+  shell: curl --head --silent http://{{ jenkins_hostname }}:8080/cli/
+  register: result
+  until: result.stdout.find("200 OK") != -1
+  retries: 18
```

Comment:
Good point. I'm actually working on a mixture of jenkins job and custom python to test idempotency of ansible runs, so I should've caught that. Your initial timeout was 300 seconds. I was bad at math and did 90 instead of 300. Do you want 60 seconds or 5 minutes as the timeout?